xdebug_var.c

```diff
@@ -905,6 +905,29 @@ xdebug_var_export_options* xdebug_var_get_nolimit_options(TSRMLS_D)
 ** Normal variable printing routines
 */
 
+static void xdebug_var_info_dump(zval *struc, xdebug_str *str, zend_bool fancy) {
+	if (fancy) {
+		xdebug_str_add(str, "<i>(", 0);
+	} else xdebug_str_add(str, "(", 0);
```

Comment:
Please put the `{` in 908 on the next line, and always add `{` … `}` for an `else` as well.